tests/test_util.py

```diff
@@ -48,34 +49,49 @@ class BufferIteratorToLineIteratorTestCase(TestCase):
 
     def test_empty(self):
         self.assertEqual(
-            list(buffer_iterator_to_line_iterator(_ for _ in ())),
+            list(yield_lines(_ for _ in ())),
             [])
 
     def test_buffered_lines(self):
         self.assertEqual(
-            list(buffer_iterator_to_line_iterator(chunk for chunk in
-                                                  [b'The quick\nbrown fox\nju',
-                                                   b'mped over\nthe lazy\ndog',
-                                                   b's.\n'])),
+            list(yield_lines(chunk for chunk in
```

Comment:
I think it's more important to maintain a consistent style, but we've been formatting new code like:

```
self.assertEqual(
    list(
        yield_lines(
            chunk for chunk in
            [
                b'The quick\nbrown fox\nju',
                b'mped over\nthe lazy\ndog', 
                b's.\n'
            ]
        )
    ),
    [b'The quick\n', b'brown fox\n', b'jumped over\n', b'the lazy\n', b'dogs.\n']
)
```
Feel free to keep formatting as-is, I don't feel strongly about it either way.